src/Model/MyParcelRequest.php

```diff
@@ -61,6 +61,7 @@ class MyParcelRequest
      */
     private $body      = '';
     private $error     = null;
+    private $errorCode = null;
```

Comment:
```suggestion
    private $errorCodes = null;
```
Omdat het meerdere zijn

Reply:
Als dit een array is, dan kan dit by default [] zijn (een lege array).

Reply:
Bij gebruik van een lege array werkte het niet meer.